src/RequestHandler/outofprocess/forwardinghandler.cpp

```diff
@@ -422,40 +426,21 @@ REQUEST_NOTIFICATION_STATUS
     if (TlsGetValue(g_dwTlsIndex) != this)
     {
         DBG_ASSERT(TlsGetValue(g_dwTlsIndex) == NULL);
-        AcquireSRWLockShared(&m_RequestLock);
+        AcquireSRWLockExclusive(&m_RequestLock);
```

Comment:
Add a comment explaining why this acquire lock is exclusive.

Reply:
done